R/annoFuse-utils.R

```diff
@@ -0,0 +1,14 @@
+
+
+
+.check_annoFuse_calls <- function(standardFusioncalls) {
+  stopifnot(is(standardFusioncalls, "data.frame"))
+  
+  # with a minimal set of columns to be there...
+  cols_fusioncalls <- c("LeftBreakpoint", "RightBreakpoint", "FusionName",
+                        "Gene1A", "Gene1B")
```

Comment:
```suggestion
  cols_fusioncalls <- c("Sample", "LeftBreakpoint", "RightBreakpoint", "FusionName",
                        "Gene1A", "Gene1B")
```

Reply:
Makes total sense, I relaxed the requirement when we were not sure whether we were enforcing - and I am happy we do so, makes things much easier also for comprehension